### DISCUSSION TITLE
Self Learning Courses not updated and therfore starmark on my certificate.

### DISCUSSION THREAD

**Discussion by k-loki** [github/india#749] [Students]
Hii,
I completed all my mandatory self learning courses on respective portals but it's still showing red on future ready talent portal.
They all are showing "in progress" eventhough i have completed them long ago.
Not a single course i completed has turned green.

Now my internship certificate has star mark, i didn't download it. should i download it?

**Comment by anupamishra333** [comment]
Hey , Is your issue Resolved?

**Reply by k-loki** [reply]
No..
I'm trying to complete the self learning course module, but it's not getting updated.
I still have the star mark on my certificate.

**Comment by anupamishra333** [comment]
Same here.  I mailed FRT but not get any positive response.

**Comment by PRIYA-S-102** [comment]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com
